docs/source/releases/index.rst

```diff
@@ -1,31 +1,14 @@
 .. dropdown:: Distribution Statement
 
- | # # # Distribution Statement A. Approved for public release. Distribution is unlimited.
- | # # #
- | # # # Author:
- | # # # Naval Research Laboratory, Marine Meteorology Division
- | # # #
- | # # # This program is free software: you can redistribute it and/or modify it under
- | # # # the terms of the NRLMMD License included with this program. This program is
- | # # # distributed WITHOUT ANY WARRANTY; without even the implied warranty of
- | # # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
- | # # # for more details. If you did not receive the license, for more information see:
- | # # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
+ | # # # This source code is protected under the license referenced at
+ | # # # https://github.com/NRLMMD-GEOIPS.
 
 .. _release_notes:
 
 *************
 Release Notes
 *************
 
-Version 1.14
-------------
-
-.. toctree::
-   :maxdepth: 1
-
-   v1_14_0a0
-
```

Comment:
Do not merge until docs/source/releases/*.rst check is failing.

Reply:
![image](https://github.com/user-attachments/assets/4c17a840-3968-467e-a3c4-a35bf69fb89c)